src/app/forms/custom-sections/trl-brakes/trl-brakes.component.ts

```diff
@@ -31,7 +31,13 @@ export class TrlBrakesComponent implements OnInit, OnChanges, OnDestroy {
   ngOnInit(): void {
     this.form = this.dfs.createForm(TrlBrakesTemplate, this.vehicleTechRecord) as CustomFormGroup;
 
-    this.form.cleanValueChanges.pipe(debounceTime(400), takeUntil(this.destroy$)).subscribe(event => this.formChange.emit(event));
+    this.form.cleanValueChanges.pipe(debounceTime(400), takeUntil(this.destroy$)).subscribe((event: any) => {
+      if (event?.axles) {
+        event.axles = (event.axles as Axle[]).filter(axle => !!axle?.axleNumber);
```

Comment:
```suggestion
        event.axles = (event.axles as Axle[]).filter(axle => axle?.axleNumber == undefined);
```
I don't know if we want to worry about 0 or not (I assume no, because you can't have an axle number 0)

Reply:
They would never be 0 as they're always normalized in tech record summary before processing.